### PR TITLE
Acceptance Test

### PR DESCRIPTION
**Pull request by OdedViner**
Signed-off-by: Oded Viner <oviner@redhat.com>

### CODE REVIEW

File: tests/managed-service/test_acceptance_managed_service.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+
+class TestAcceptanceMS(ManageTest):
+    """
+    Acceptance test Managed Service
+
+    """
+
+    @managed_service_required
+    def test_acceptance_managed_service(
```
Comment (ebenahar) [inline tests/managed-service/test_acceptance_managed_service.py:19]: isn't this test case applicable for non MS clusters too?

Comment (OdedViner) [reply tests/managed-service/test_acceptance_managed_service.py:19]: This test applicable for non-MS clusters too

**Comment by jilju** [comment]
LGTM

File: tests/managed-service/test_acceptance.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+                    )
+                    break
+                logger.info(
+                    f"Expanded size of PVC {pod_obj.pvc.name} is not reflected"
+                    f" on pod {pod_obj.name}. New size on mount is not "
+                    f"{pvc_size_new}G as expected, but {new_size_mount}. "
+                    f"Checking again."
+                )
+        logger.info(
+            f"Verified: Modified size {pvc_size_new}G is reflected " f"on all pods."
```
Comment (fbalak) [inline tests/managed-service/test_acceptance.py:183]:
```suggestion
            f"Verified: Modified size {pvc_size_new}G is reflected "on all pods."
```

File: tests/managed-service/test_acceptance.py [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+            writes = fio_result.get("jobs")[0].get("write").get("iops")
+            logger.info(f"Read: {reads}")
+            logger.info(f"Write: {writes}")
+
+        self.clone_pvc(teardown_factory=teardown_factory)
+        self.resize_pvc(pvc_size_new=3)
+
+    def clone_pvc(self, teardown_factory):
+        """
+        Clone PVC
```
Comment (fbalak) [inline tests/managed-service/test_acceptance.py:107]: Please add args docstring

File: tests/managed-service/test_acceptance.py (hunk 2 above)
Comment (OdedViner) [reply tests/managed-service/test_acceptance.py]: done

File: tests/managed-service/test_acceptance.py [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+            ),
+            (
+                constants.CEPHBLOCKPOOL,
+                constants.ACCESS_MODE_RWO,
+                constants.VOLUME_MODE_BLOCK,
+            ),
+            (
+                constants.CEPHBLOCKPOOL,
+                constants.ACCESS_MODE_RWX,
+                constants.VOLUME_MODE_BLOCK,
```
Comment (ebenahar) [inline tests/managed-service/test_acceptance.py:52]: can we add one more permutation of raw_block_pv?

**Review by jilju — APPROVED** [review]

**Review by fbalak — APPROVED** [review]

File: tests/managed-service/test_acceptance.py [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+logger = logging.getLogger(__name__)
+
+
+class TestAcceptanceMS(ManageTest):
+    """
+    Acceptance test Managed Service
+
+    """
+
+    @acceptance
```
Comment (fbalak) [inline tests/managed-service/test_acceptance.py:18]: Please add managed_service_required marker.

**Review by fbalak — CHANGES_REQUESTED** [review]

**Pull request closed without merging**